lib/api/dsl/operators.js

```diff
@@ -177,7 +177,7 @@ module.exports = operators = {
    * Return true if the document field is into the given bounding box
    *
    * @param {String} field the field that we have to check
-   * @param {String} value containing the bounding box coords
+   * @param {String|{lat: *, lon: *}|{from: *, to: *}} value containing the bounding box coords
```

Comment:
If I read the code correctly, then `value` can never be of type `string`, it was a JSDoc error.
So I guess the correct typing description should be: 
`@param {{lat: *, lon: *}|{from: *, to: *}} value containing the bounding box coords`